cyral/resource_cyral_repository_binding.go

```diff
@@ -56,6 +57,11 @@ func resourceRepositoryBinding() *schema.Resource {
 				Optional: true,
 				Default:  "0.0.0.0",
 			},
+			"select_sidecar_as_idp_access_gateway": {
```

Comment:
```suggestion
			"sidecar_as_idp_access_gateway": {
```

As the variable is a boolean already, `sidecar_as_idp_access_gateway` is easy to understand and shorter. @VictorGFM , lmk your thoughts on this one. 
BTW: the actual API field name is not good and we should refactor in the future, deprecating the old name.

Reply:
I agree with you, thanks for the suggestion.